src/deepsparse/tasks.py

```diff
@@ -134,3 +142,11 @@ def is_yolo(cls, task: str) -> bool:
         :return: True if it is an segmentation task using YOLO, False otherwise
         """
         return cls.yolo.yolo.matches(task)
+
+    @classmethod
+    def task_names(cls):
+        task_names = []
+        for task_category in cls.all_task_categories:
+            for task in task_category:
+                task_names += [task._name] + task._aliases
```

Comment:
can we add a check here to add aliases to task_names only if they're different from task._name?